rotkehlchen/chain/ethereum/modules/aave/graph.py

```diff
@@ -247,7 +246,7 @@ def _parse_atoken_balance_history(
             )
             continue
 
-        reserve_address = to_checksum_address('0x' + pairs[2])
+        reserve_address = deserialize_ethereum_address('0x' + pairs[2])
```

Comment:
you need to handle the possibilitiy of a `DeserializationError` here. I guess log.error() and `continue like is done with the `asset` below is fine.